apps/prairielearn/elements/pl-drawing/pl-drawing.py

```diff
@@ -217,7 +217,7 @@ def render_drawing_items(elem, curid=0, defaults=None):
     for el in elem:
         if el.tag is lxml.etree.Comment:
             continue
-        elif el.tag == "pl-drawing-group":
+        if el.tag == "pl-drawing-group":
```

Comment:
I'm on the fence about this type of change. I slightly lean towards enforcing it, it reduces confusion as to what needs to be checked for me.

Reply:
I'm fine with this. I lean toward putting a blank line after `continue` and before the rest of the code, but this is purely an aesthetic preference.